client/src/components/view-job-posts-component/job-type-chips-component/index.jsx

```diff
@@ -10,6 +10,11 @@ const chipColors = {
   Seasonal: { border: "#00838F", bg: "#E0F7FA", text: "#000" }, // Teal
   "Part-time": { border: "#EC407A", bg: "#FCE4EC", text: "#000" }, // Pink
   Internship: { border: "#3F51B5", bg: "#E8EAF6", text: "#000" }, // Indigo
+  Inactive: { border: "#9E9E9E", bg: "#D3D3D3", text: "#000" }, // Gre
+  Draft: { border: "#1976D2", bg: "#E3F2FD", text: "#0D47A1" }, // Blue
```

Comment:
We've recently updated the colours for the chips, can you please change accordingly? 
![image](https://github.com/user-attachments/assets/5e28a708-4f52-4631-9f0d-263f10ee7c74)